### DISCUSSION TITLE
Is there a way to get information of merchant fulfillment product?

### DISCUSSION THREAD

**Discussion by hopeko** [abuzuhri/Amazon-SP-API-CSharp#71] [Q&A] [answered]
I can get sku, asin, productName, inventory with `FbaInventory.GetInventorySummaries()`
Is there a way to get similar information of merchant fulfillment products?

**Comment by abuzuhri** [comment]
> I can get sku, asin, productName, inventory with `FbaInventory.GetInventorySummaries()`
> 
> Is there a way to get similar information of merchant fulfillment products?

Hello 

You can request report there is lite report for seller inventory

**Comment by abuzuhri** [comment]
check this report
```csharp 
amazonConnection.Reports.CreateReportAndDownloadFile(ReportTypes.GET_MERCHANT_LISTINGS_DATA_LITE);
```

**Reply by hopeko** [reply]
Thanks! There're seller-sku, quantity, price, product-id columns in the report, but no productName. How can I get it?

**Comment by abuzuhri** [comment, marked as answer]
check this page for all report and details for that report
https://github.com/amzn/selling-partner-api-docs/blob/main/references/reports-api/reporttype-values.md

maybe you need this report GET_MERCHANT_LISTINGS_ALL_DATA

**Reply by hopeko** [reply]
> check this page for all report and details for that report https://github.com/amzn/selling-partner-api-docs/blob/main/references/reports-api/reporttype-values.md
> 
> maybe you need this report GET_MERCHANT_LISTINGS_ALL_DATA

It returns all product data (including AFN products). Is it a bug? I try GET_MERCHANT_LISTINGS_DATA too, the same.

**Reply by abuzuhri** [reply]
Not it's not bug this report return all listing and you can filter what you need
